cheroot/workers/threadpool.py

```diff
@@ -108,7 +108,7 @@ def run(self):
 
         Retrieves incoming connections from thread pool.
         """
-        self.server.stats['Worker Threads'][self.getName()] = self.stats
+        self.server.stats['Worker Threads'][self.get_name()] = self.stats
```

Comment:
I'm pretty sure it's
```suggestion
        self.server.stats['Worker Threads'][self.name] = self.stats
```